common/components/deploy_model/spec.yaml

```diff
@@ -0,0 +1,187 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+name: deploy_model
+version: 0.0.1
+type: command
+
+is_deterministic: True
+
+display_name: Deploy model
+description:
+  Deploy a model to a workspace. The component works on compute with [MSI](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-create-manage-compute-instance?tabs=python) attached.
+
+environment: azureml:python-sdk-v2:1
```

Comment:
Image for this environment was never released to MCR

Reply:
Has the release completed, @novaturient95 ?

Reply:
Release is done. Approving.